includes/Core/CLI/Authentication_CLI_Command.php

```diff
@@ -2,52 +2,54 @@
 /**
  * Site Kit Authentication CLI Commands
  *
- * @package   Google\Site_Kit
- * @copyright 2019 Google LLC
+ * @package   Google\Site_Kit\Core\CLI
+ * @copyright 2020 Google LLC
  * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://sitekit.withgoogle.com
  */
 
-namespace Google\Site_Kit\CLI;
+namespace Google\Site_Kit\Core\CLI;
 
-use Google\Site_Kit\Context;
 use Google\Site_Kit\Core\Storage\User_Options;
 use Google\Site_Kit\Core\Authentication\Authentication;
-use WP_CLI;
-use WP_CLI_Command;
-
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly.
-}
-
-WP_CLI::add_command( 'google-site-kit auth', __NAMESPACE__ . '\Authentication_CLI_Command' );
 
 /**
- * Site Kit Authentication CLI Command
+ * Site Kit authentication commands
```

Comment:
This description is what is shown in the CLI help so we should word this a bit differently.
```suggestion
 * Manages Site Kit user authentication for Google APIs.
```

Reply:
Updated.